libraries/scripting/dependencies/src/kotlin/script/experimental/dependencies/resolverNamedOptions.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package kotlin.script.experimental.dependencies
+
+fun ExternalDependenciesResolver.Options.value(name: DependenciesResolverOptionsName) =
+    value(name.key)
+
+fun ExternalDependenciesResolver.Options.flag(name: DependenciesResolverOptionsName) =
+    flag(name.key)
+
+operator fun MutableMap<String, String>.set(key: DependenciesResolverOptionsName, value: String) {
+    put(key.key, value)
+}
+
+
+enum class DependenciesResolverOptionsName {
+    TRANSITIVE,
+    DEPENDENCY_SCOPES;
```

Comment:
Using underscores seems unusual for the maven configuration, and in maven this parameter called just `scope`, so I'd rename it, so the people can guess the param name easier.
If we want to have the naming flexibility - I'd add the string name parameter to the enum (and use `toLowerCase` as a default value).

Reply:
Fixed this name and added a parameter